src/libxtp/openmp_cuda.cc

```diff
@@ -465,7 +487,8 @@ Eigen::MatrixXd OpenMP_CUDA::getReductionVar() {
   for (Index i = 0; i < Index(gpus_.size()); i++) {
     GPU_data& gpu = gpus_[i];
     gpu.activateGPU();
-    cpus_[i].reduce() = *(gpu.temp.back());
+    Eigen::MatrixXd temp=*(gpu.temp.back());
```

Comment:
Out of curiosity why do you need this temp array here?

Reply:
Good point I do not :)